### DISCUSSION TITLE
mongo db find with  array size

### DISCUSSION THREAD

**Discussion by bwp618** [mikro-orm/mikro-orm#3065] [Q&A]
db.collection.find( { field: { $size: 1 } } );

How can I execute the above statement?

**Comment by B4nan** [comment]
You can access the native mongo collection via `em.getCollection()` (the `em` needs to be properly typed to the EM expoerted from `@mikro-orm/mongodb` package). The result will be raw, if you want to convert it to entities, you can use `em.map()`.

**Reply by bwp618** [reply]
MikroOrmModule.forRootAsync({
      useFactory: async (config: ConfigService) => {
        return ({
          debug: true,
          ...
        })
      },
    }),

em.getCollection is not working with debug option 
is it right?
